extras/openvino-operator-openshift/openshift-bundle/0.2.0/manifests/openvino-operator.clusterserviceversion.yaml

```diff
@@ -226,7 +227,7 @@ spec:
                   - --metrics-bind-address=127.0.0.1:8080
                   - --leader-elect
                   - --leader-election-id=ovms-operator
-                image: registry.connect.redhat.com/intel/openvino-operator:0.2.0
+                image: registry.connect.redhat.com/intel/ovms-operator:0.2.0
```

Comment:
Is this change intentional? Shouldn't it be `openvino-operator`?

Reply:
@mzegla it is intentional, otherwise we would need to create new offering in RH catalogue. It was agreed to do it for golang version later